trusted-types/Element-setAttribute-not-enforced-in-non-TT-documents-globals-CSP.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
```

Comment:
Once the relevant spec PR (https://github.com/whatwg/dom/pull/1268) is merged, the relevant spec should be linked here via `<link rel="help" href="<linkToSpec>">`.